libraries/AC_Planck/AC_Planck.h

```diff
@@ -95,9 +120,31 @@ class AC_Planck {
     uint32_t timestamp_ms = 0;
   }_status;
 
+  enum planck_ack_status
+  {
+    PLANCK_ACK,
+    PLANCK_NACK,
+    PLANCK_WAITING_FOR_ACK
+  } _ack_status = PLANCK_WAITING_FOR_ACK;
+
+  uint32_t _last_cmd_req_t_ms = 0;
+  uint32_t _last_ack_t_ms = 0;
+  uint32_t _next_gcs_message_t_ms = 0;
+  uint32_t _gcs_msg_wait_t_ms = 1000;
+  int16_t _last_cmd_req_id = -1;
```

Comment:
I think `_last_cmd_req_id`, `_last_cmd_req_t_ms`, `_last_ack_t_ms`, and `_ack_status` should all be part of a class/struct